newspaper/extractors.py

```diff
@@ -504,7 +504,7 @@ def get_meta_data(self, doc):
             key_head = key.pop(0)
             ref = data[key_head]
 
-            if isinstance(ref, str):
+            if isinstance(ref, str) or isinstance(ref, int):
```

Comment:
I wonder if the `ref / ref = data[key_head]` object can be of any other type besides `str` or `int`? If a float appears the same bug would happen correct?